b2sdk/raw_api.py

```diff
@@ -143,7 +144,14 @@ def get_download_authorization(
         pass
 
     @abstractmethod
-    def get_file_info(self, api_url, account_auth_token, file_id):
+    def get_file_info_by_id(self, api_url: str, account_auth_token: str,
+                            file_id: str) -> Dict[str, Any]:
+        pass
+
+    @abstractmethod
+    def get_file_info_by_name(
```

Comment:
In `bucket.py` and `session.py` you use `bucket_id` but here you have `bucket_name`.

This is a direct cause of the unit tests failure:
```
self = <b2sdk.raw_simulator.RawSimulator object at 0x7f3fac713c50>, api_url = 'http://api.example.com', account_auth_token = 'auth_token_0', bucket_name = 'bucket_0', file_name = 'a'

    def get_file_info_by_name(self, api_url, account_auth_token, bucket_name, file_name):
>       bucket = self.bucket_name_to_bucket[bucket_name]
E       KeyError: 'bucket_0'

b2sdk/raw_simulator.py:942: KeyError
```